### PR TITLE
BACKLOG-12863 : fix login issue

### PR DESCRIPTION
**Pull request by dgriffon**
https://jira.jahia.org/browse/BACKLOG-12863

### CODE REVIEW

File: src/main/resources/root.jsp [hunk 1]
```diff
@@ -17,6 +17,23 @@
...
+            contextPath: '${contextPath}',
+            user: {
+                username: '${currentUser.name}',
+                fullname: '${user:fullName(currentUser)}',
+                email: '${currentUser.properties['j:email']}',
+                path: '${currentUser.localPath}'
+            },
+            config: ${config},
+            siteKey: '${defaultSite.siteKey}',
+            siteUuid: '${defaultSite.identifier}',
```
Comment (tdraier) [inline src/main/resources/root.jsp]: what do you need the siteUuid for ?

File: src/main/resources/root.jsp [hunk 2]
```diff
@@ -17,6 +17,23 @@
 
     <internal:gwtGenerateDictionary/>
     <internal:gwtInit/>
+
+    <script type="text/javascript">
```
Comment (tdraier) [inline src/main/resources/root.jsp]: why moved the code here ?

File: src/main/resources/root.jsp (hunk 1 above)
Comment (dgriffon) [reply src/main/resources/root.jsp]: for GWT to set the site correctly

File: src/main/resources/root.jsp (hunk 2 above)
Comment (dgriffon) [reply src/main/resources/root.jsp]: because GWT needs it to get initialised properly

File: src/main/resources/root.jsp (hunk 1 above)
Comment (tdraier) [reply src/main/resources/root.jsp]: the site should anyway be set by the redux listener

**Pull request merged**